README.md

```
@@ -666,11 +666,11 @@ The `package.lisa.json` strategy provides a specialized approach to managing `pa
 
 ### How It Works
 
-Each project type directory (all/, typescript/, expo/, nestjs/, cdk/, npm-package/) can contain a `package.lisa.json` file in the `tagged-merge/` subdirectory:
+Each project type directory (all/, typescript/, expo/, nestjs/, cdk/, npm-package/) can contain a `package.lisa.json` file in the `package-lisa/` subdirectory:
 
 ```
```

markdownlint-cli2 (0.20.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)